hooks/post-turn-quality-stop-hook.py

```
@@ -0,0 +1,447 @@
+    return "no makefile found" in lowered
+
+
+def get_make_targets(repo: Path) -> tuple[set[str] | None, str | None]:
+    try:
+        p = run(["make", "-qp", "--no-print-directory"], repo)
+    except FileNotFoundError:
+        return None, "make not found on PATH"
+
+    # make -q can return 0 or 1 without being an error; 2 means failure
+    if p.returncode == 2:
+        combined = "\n".join([p.stderr.strip(), p.stdout.strip()]).strip()
```

Ruff (0.14.14)

[error] Consider `f"{p.stderr.strip()}\n{p.stdout.strip()}"` instead of string join

Replace with `f"{p.stderr.strip()}\n{p.stdout.strip()}"`

(FLY002)